logdevice/server/ServerSettings.cpp

```diff
@@ -582,6 +582,14 @@ void ServerSettings::defineSettings(SettingEasyInit& init) {
      SERVER | REQUIRES_RESTART,
      SettingsCategory::Configuration)
 
+    ("use-client-provided-tos",
```

Comment:
For consistency with the other DSCP settings, I'd prefer if we name this setting `enable-dscp-reflection`.